specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/iotdps.json

```diff
@@ -0,0 +1,2622 @@
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "operationId": "IotDpsResource_ListByResourceGroup",
+        "description": "Get a list of all provisioning services in the given resource group.",
+        "x-ms-examples": {
+          "DPSListByResourceGroup": {
+            "$ref": "./examples/DPSListByResourceGroup.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Resource group identifier."
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of provisioning services in the resource group.",
+            "schema": {
+              "$ref": "#/definitions/ProvisioningServiceDescriptionListResult"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/operationresults/{operationId}": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "operationId": "IotDpsResource_GetOperationResult",
+        "description": "Gets the status of a long running operation, such as create, update or delete a provisioning service.",
+        "x-ms-examples": {
+          "DPSGetOperationResult": {
+            "$ref": "./examples/DPSGetOperationResult.json"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Operation id corresponding to long running operation. Use this to poll for the status."
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Resource group identifier."
+          },
+          {
+            "name": "provisioningServiceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of provisioning service that the operation is running on."
+          },
+          {
+            "name": "asyncinfo",
```

Comment:
Why is this needed? The GET operationsresult URL should be sufficient to get the async operation state.

Reply:
Temporarily resolved. See details in this [comment](https://github.com/Azure/azure-rest-api-specs/pull/12378#discussion_r560402312)